src/docs/content/docs/download/docker.md

```diff
@@ -13,28 +13,99 @@ parent = "download"
 
 Building Reaper packages requires quite a few dependencies, especially when making changes to the web interface code. In an effort to simplify the build process, Dockerfiles have been created that implement the build actions required to package Reaper.
 
-To build the JAR and other packages which are then placed in the _packages_ directory run the following commands from the top level directory:
+To build the JAR and other packages which are then placed in the _src/packages_ directory run the following commands:
 
 ```bash
-docker-compose -f src/packaging/docker-build/docker-compose.yml build
-docker-compose -f src/packaging/docker-build/docker-compose.yml run build
+cd src/packaging/docker-build
+docker-compose build
+docker-compose run build
 ```
 
 ## Building Reaper Docker Image
 
-To build the Reaper Docker Image which is then added to the local image cache using the `cassandra-reaper:latest` tag, run the following commands from the top level directory:
+### Prerequisite
+
+The generation of the Docker image requires that the JAR file be built and placed in the _src/packages_ directory. If the JAR package is missing from the directory then it can built using either the steps in the [Docker package build](building-reaper-packages-with-docker) section (above), or in the <a href="../building#building-jars-from-source">Building from Source</a> section. 
```

Comment:
@ossarga can you do a proper Markdown link rather than href link?

Reply:
Unsure how to link across to a different section in the Markdown.

Reply:
[Building from Source](../building#building-jars-from-source) should work.

Reply:
Hmmm ok. I probably specified it wrong when I was trying it. I have changed the link to use the markdown recommended in the Hugo docs
```
[Building from Source]({{< relref "building.md#building-jars-from-source" >}})
```